docs/src/main/sphinx/connector/teradata.md

```diff
@@ -0,0 +1,329 @@
+# Teradata connector
+
+```{raw} html
+<img src="../_static/img/teradata.png" class="connector-logo">
+```
+
+The Teradata connector allows querying and creating tables in an external Teradata database.
```

Comment:
Wrap at 80 characters. Same for other places.

Reply:
addressed in new PR https://github.com/trinodb/trino/pull/26731